index.js

```diff
@@ -9,27 +9,26 @@ import * as path from 'path';
 function koaDevware (compiler, options) {
   const dev = devMiddleware(compiler, options);
 
-  function middleware (context, next) {
+  function waitUntilValid() {
```

Comment:
this isn't an accurate name for the method, as it also rejects on failure, which is an invalid state